src/datachain/sql/types.py

```diff
@@ -439,6 +506,50 @@ def binary(self):
         return None
 
 
+class DBDefaults:
```

Comment:
Use default values in `CREATE TABLE` query based on DB implementations.
It is used in Studio, see related PR.